src/styles/_catalog.scss

```diff
@@ -1,22 +1,28 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
-  max-width: 944px;
+  display: grid;
+  gap: 48px;
+  justify-content: center;
   margin: 0 auto;
+  grid-template-columns: repeat(1, $cardWidth);
 
   &__card {
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    width: 200px;
+    display: grid;
+    place-items: center;
+    width: $cardWidth;
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
-    margin-bottom: 48px;
   }
 
-  &__card:not(:nth-child(4n)) {
-    margin-right: 48px;
+  @media (min-width: 488px) {
```

Comment:
Generally good practice is to use whole styling properties for each class in one place. So those breakpoints are for `.catalog` class so move them up before `&__card` and not behind as you do now